application/src/main/java/org/togetherjava/tjbot/commands/mediaonly/MediaOnlyChannelListener.java

```diff
@@ -37,27 +36,24 @@ public void onMessageReceived(MessageReceivedEvent event) {
             return;
         }
 
-        if (messageHasNoMediaAttached(event)) {
-            deleteMessage(event).flatMap(any -> dmUser(event)).queue();
+        Message message = event.getMessage();
+        // Checks if it's a thread creation message. No need to do anything in that case then
+        if (message.toString().contains("THREAD_CREATED")) {
```

Comment:
Thats not the proper JDA way to test this. Also, in general, never do any business logic based on `toString()`, this method may change its output at any time.

Instead, check the type of the message `message.getType() == ...`. As an example, see the class that cleans up bot messages in the help channel, it can filter thread creation messages already.